pkg/framework/plugins/defaultevictor/defaultevictor_test.go

```diff
@@ -347,225 +651,6 @@ func TestDefaultEvictorFilter(t *testing.T) {
 			evictSystemCriticalPods: true,
 			priorityThreshold:       &lowPriority,
 			result:                  true,
-		}, {
```

Comment:
Worth introducing a new test that has `nodeFit: true` to prove setting the arg has no effect on the `Filter` ep functionality.